bootstrap.sh

```diff
@@ -37,51 +38,8 @@ fi
 
 cd $BASE_DIR
 
-SCRIPT_JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64/"
-export JAVA_HOME="${SCRIPT_JAVA_HOME}"
+(cd $BASE_DIR; ./scripts/bootstrap-BenchmarkJava.sh $BOOTSTRAP_OPTIONS)
+(cd $BASE_DIR; ./scripts/bootstrap-BenchmarkJava-mutated.sh$BOOTSTRAP_OPTIONS)
+(cd $BASE_DIR; ./scripts/bootstrap-reality-check.sh$BOOTSTRAP_OPTIONS)
```

Comment:
spaces

Reply:
This is a trick. `BOOTSTRAP_OPTIONS` is empty by default, and its updates add a space and an option after, so the options are either `" or `" *options* *option* ... *option*". The additional space seems redundant, doesn't it?

Reply:
I agree. It can become an issue eventually.